CONTRIBUTING.md

```diff
@@ -40,6 +40,14 @@ If you are proposing a feature:
 -   Remember that this is a volunteer-driven project, and that contributions are welcome
     :)
 
+## Dev environment setup
+
+We use some tools as part of our development workflow which you'll need to install into
+your host environment:
+
+-   [poetry](https://python-poetry.org/) for packaging and dependency management
+-   [poethepoet](https://github.com/nat-n/poethepoet) for running development tasks
```

Comment:
Do you really need to install this? Isn't it installed automatically by poetry?

Reply:
It's possible to have poe installed inside the poetry managed environment, and use it with poetry run/shell, but I've not added it as a dependency because I think it's more useful this way.

Having poe installed in the host environment (such as via pip, or pipx) is also necessary for tab completion to work.